### PR TITLE
New Data Source: 'azurerm_eventhub_consumer_group'

### PR DESCRIPTION
**Pull request by brunhil**
Add new data source 'azurerm_eventhub_consumer_group'

=== RUN   TestAccDataSourceAzureRMEventHubConsumerGroup_complete
=== PAUSE TestAccDataSourceAzureRMEventHubConsumerGroup_complete
=== CONT  TestAccDataSourceAzureRMEventHubConsumerGroup_complete
--- PASS: TestAccDataSourceAzureRMEventHubConsumerGroup_complete (202.86s)
PASS
ok      github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/eventhub/tests   
        203.125s

### CODE REVIEW

File: azurerm/internal/services/eventhub/data_source_eventhub_consumer_group.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		return fmt.Errorf("Error: EventHub Consumer Group %s: %+v", name, err)
+	}
+
+	d.SetId(*resp.ID)
+
+	d.Set("name", name)
+	d.Set("eventhub_name", eventHubName)
+	d.Set("namespace_name", namespaceName)
+	d.Set("resource_group_name", resourceGroup)
+	d.Set("user_metadata", resp.ConsumerGroupProperties.UserMetadata)
```
Comment (katbyte) [inline azurerm/internal/services/eventhub/data_source_eventhub_consumer_group.go]: we'll need to check @ConsumerGroupProperties is not nil here

File: website/docs/d/eventhub_consumer_group.html.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  name                = "${azurerm_eventhub_consumer_group.test.name}"
+  namespace_name      = "${azurerm_eventhub_namespace.test.name}"
+  eventhub_name       = "${azurerm_eventhub.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
```
Comment (katbyte) [inline website/docs/d/eventhub_consumer_group.html.markdown]: This should be
```suggestion
The following arguments are exported:
```

File: website/docs/d/eventhub_consumer_group.html.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+* `name` - (Required) Specifies the name of the EventHub Consumer Group resource.
+
+* `namespace_name` - (Required) Specifies the name of the grandparent EventHub Namespace.
+
+* `eventhub_name` - (Required) Specifies the name of the EventHub.
+
+* `resource_group_name` - (Required) The name of the resource group in which the EventHub Consumer Group's grandparent Namespace exists.
```
Comment (katbyte) [inline website/docs/d/eventhub_consumer_group.html.markdown]: we don't need required/optional on data source documentation, and could we use language consistent with the rest of the provider?

**Comment by tombuildsstuff** [comment]
@brunhil hope you don't mind but I've pushed the rebase for this

**Comment by tombuildsstuff** [comment]
Test still passes:

Screenshot 2020-02-11 at 15 58 28

**Pull request merged**